Expensify/App
# Make sure users are signed in when clicking on managed cards

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/171698

### Tests
- Create a workspace via the global create button and give it a name
- Make sure you are signed out of expensify.com and staging.expensify.com 
- Go to the workspace settings and click on manage cards:
![image](https://user-images.githubusercontent.com/521248/127185587-f0bb62f5-648d-4768-9b5e-ebc44aae0920.png)
- Check that a new tab opens and it's in the https://www.expensify.com/domain_companycards (or https://staging.expensify.com/domain_companycards) and that you are logged in to the same account that in e.cash

### Tested On
